### PR TITLE
Prepare Epicea for latest Spec2

### LINKED ISSUE #5056: Epicea: prepare for latest Spec2
As Pavel reported in a private email:
"Epicea window that should allow you to confirm the changes you want to apply does not show the related confirmation button. "

I could reproduce it in a today's P8 + evaluation of:
~~~
Metacello new
        githubUser: 'pharo-spec' project: 'Spec' commitish: 'master' path: 'src';
        baseline: 'Spec2';
        onConflict: [ :e | e useIncoming ];
        onUpgrade: [ :e | e useIncoming ];
        ignoreImage;
        load
~~~

And am exploring a solution.

### PR DESCRIPTION
**Pull request by tinchodias**
Fixes #5056.

### CODE REVIEW

File: src/EpiceaBrowsers/EpLogBrowserPresenter.class.st [hunk 1]
```diff
@@ -34,15 +33,12 @@ EpLogBrowserPresenter class >> defaultLog [
 { #category : #specs }
 EpLogBrowserPresenter class >> defaultSpec [
 	^ SpPanedLayout newVertical
-		position: 60 percent;
+		position: 0.6;
 		add: #itemsPresenter;
```
Comment (tinchodias) [inline src/EpiceaBrowsers/EpLogBrowserPresenter.class.st:37]: This wasn't on purpose, 60 percent looks better...

File: src/EpiceaBrowsers/EpLogBrowserPresenter.class.st [hunk 2]
```diff
@@ -551,30 +547,27 @@ EpLogBrowserPresenter >> initialize [
...
+
+	self beHistoryMode.
 ]
 
 { #category : #initialization }
 EpLogBrowserPresenter >> initializeWindow: aWindowPresenter [
-	aWindowPresenter
-		initialExtent: 550 @ 700;
-		title: 'Epicea - Log Browser'
+	aWindowPresenter initialExtent: 550 @ 700
```
Comment (tinchodias) [inline src/EpiceaBrowsers/EpLogBrowserPresenter.class.st:570]: title is overriden by beHistory and 2 friends so no need to have it here.

File: src/EpiceaBrowsers/EpLogBrowserPresenter.class.st [hunk 3]
```diff
@@ -647,7 +640,7 @@ EpLogBrowserPresenter >> menuMorphForSelectedItems [
 { #category : #accessing }
 EpLogBrowserPresenter >> monitor [
 	
-	^ monitor ifNil: [ EpMonitor current ]
+	^ EpMonitor current
 ]
```
Comment (tinchodias) [inline src/EpiceaBrowsers/EpLogBrowserPresenter.class.st:644]: monitor slot was always nil (there is no setter) so I removed it

File: src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st [hunk 4]
```diff
@@ -9,18 +9,14 @@ Class {
 		'filtersButton',
 		'statusLabel'
 	],
-	#classInstVars : [
-		'refresh'
-	],
```
Comment (tinchodias) [inline src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st:14]: removed this unused slot

File: src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st [hunk 5]
```diff
@@ -52,13 +50,8 @@ EpLogBrowserToolbarPresenter >> beRevertMode [
 		action: [ logBrowserPresenter revertAllChangesAndCloseWindow ]
 ]
 
-{ #category : #private }
-EpLogBrowserToolbarPresenter >> filtersAction [
-	logBrowserPresenter filtersSubMenu openWithSpecAt: ActiveHand position
-]
```
Comment (tinchodias) [inline src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st:58]: decided to inline `filtersAction`, looks better and more consistent in the class.

File: src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st [hunk 6]
```diff
@@ -52,13 +50,8 @@ EpLogBrowserToolbarPresenter >> beRevertMode [
...
-
 { #category : #initialization }
-EpLogBrowserToolbarPresenter >> initializeWidgets [
+EpLogBrowserToolbarPresenter >> initializePresenters [
```
Comment (tinchodias) [inline src/EpiceaBrowsers/EpLogBrowserToolbarPresenter.class.st:54]: renamed several of those since Spec2 suggests it

**Comment by tinchodias** [comment]
IMPORTANT NOTE:

Didn't check if it works with current Spec2, only tested it after loading latest.

**Comment by pavel-krivanek** [comment]
Cool! Thank you, the changes look harmless to the current version.

**Comment by pavel-krivanek** [comment]
I tested it and the problematic functionality that was not working in the very latest Spec works fine in the current one even after the fix. Good job!

**Review by pavel-krivanek — APPROVED** [review]

**Comment by tinchodias** [comment]
Great! 
the bottom border of the button was hidden/clipped but I think it's something that Spec/Morphic will fix independently.

**Pull request merged**